core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -72,15 +72,15 @@
     // For future use: re-send a failed transmission back to the dispatcher
     private TransmissionDispatcher transmissionDispatcher;
 
-    private String serverUri;
+    private final String serverUri;
 
-    private volatile boolean stopped;
```

Comment:
this is thread-safe.  did intellij inspection fail in this case?

Reply:
if it can be changed to local var, then that solves the thread-safety issue 😄

Reply:
volatile or local var?

Reply:
local vars have no thread-safety problems, since they are not shared across threads

Reply:
do we need stopped at all?  i don't see that it becomes true anywhere in the class...  final does not make sense to stopped, it should be modifiable after the initial value.

Reply:
👍 good point, removed